torch/distributed/internal_rpc_utils.py

```diff
@@ -99,11 +99,13 @@ def deserialize(self, binary_data, tensor_table):
 _internal_rpc_pickler = _InternalRPCPickler()
 
 
-def run_python_udf_internal(pickled_python_udf, tensors):
+def _python_udf_run(pickled_python_udf, tensors):
     r"""
-    Internal python function will be imported and executed in C++ land
-    it unpickles pickled python udf strings and tensors and run the python
-    udf, return serialized result and tensor tables
+    This function is exclusively called from C++.
+    See ``torch/csrc/distributed/rpc/python_rpc_handler.cpp``.
+
+    Unpickles a Python function and its arguments, runs the function,
+    and pickles and returns the function's return value.
     """
     python_udf = _internal_rpc_pickler.deserialize(pickled_python_udf, tensors)
```

Comment:
The name "_python_udf_run" doesn't indicate it's actually taking a serialized python function with args, deserialize it, run it, and serialize it.

Should we call it something else?

Reply:
_python_udf_remote_run?

Reply:
Let's try to avoid using `remote` here, as `dist.remote` is another API with a different scope. 

In the message level, we have `PYTHON_CALL` and `PYTHON_RET`. Shall we use a similar name here? For example:

* `_python_udf_run` -> `_run_serialized_python_call`
* `load_python_udf_result_internal ` -> `_deserialize_python_ret`

Reply:
Good points. I stuck with the original naming because I didn't know about the intent here. As it can be any Python function, I think @mrshenli's suggestions are good.